crypto/src/lib.rs

```diff
@@ -18,6 +18,7 @@ mod note_payload;
 mod nullifier;
 mod prf;
 pub mod proofs;
+mod swap;
```

Comment:
This could be `pub mod swap;`, without the `pub use` below, if we wanted to isolate the (more niche) swap functionality into one submodule.